pkg/docker/config/config_test.go

```diff
@@ -110,13 +110,14 @@ func TestGetAuth(t *testing.T) {
 
 	for _, configPath := range configPaths {
 		for _, tc := range []struct {
-			name             string
-			hostname         string
-			path             string
-			expectedUsername string
-			expectedPassword string
-			expectedError    error
-			sys              *types.SystemContext
+			name                  string
+			hostname              string
+			path                  string
+			expectedUsername      string
+			expectedPassword      string
+			expectedError         error
+			expectedIdentityToken string
```

Comment:
Non-blocking: This could be an `expected types.DockerAuthConfig` now. Also, the “error” is conceptually separate from username+password+token, and should be the last of the four.